openeo/udp.py

```diff
@@ -0,0 +1,34 @@
+from abc import ABC, abstractmethod
+
+
+class ProcessGraph(ABC):
```

Comment:
Are we going to use this base class in other places than only for `RESTProcessGraph`? If not, it seems unnecessary abstraction and duplication to maintain (e.g. doc strings)